dmutils/external.py

```diff
@@ -5,17 +5,32 @@
 # Buyer frontend
 
 
+@external.route('/')
+def index():
+    raise NotImplementedError()
+
+
 @external.route('/<framework_framework>/opportunities/<brief_id>')
 def get_brief_by_id(framework_framework, brief_id):
     raise NotImplementedError()
 
 
+@external.route('/g-cloud/suppliers')
+def suppliers_list_by_prefix():
+    raise NotImplementedError()
+
+
 @external.route('/help')
 def help():
     raise NotImplementedError()
 
+
 # Supplier frontend
 
+@external.route('/suppliers')
+def dashboard():
```

Comment:
We should probably call this `supplier_dashboard`.

Reply:
done @samuelhwilliams